ui/job-view/CustomJobActions.jsx

```diff
@@ -131,7 +131,7 @@ class CustomJobActions extends React.PureComponent {
     let input = null;
     if (validate && payload) {
       try {
-        input = jsyaml.safeLoad(payload);
+        input = jsyaml.load(payload);
```

Comment:
I assume there is something in the payload which is not supported in safeLoad

Reply:
`safeLoad` got dropped from the v4 implementation of the package.